cmd/cmd_util.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright © 2021 The Persistent-Volume-Migrator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"persistent-volume-migrator/internal/ceph/rbd"
+	"persistent-volume-migrator/internal/kubernetes"
```

Comment:
naming the package `kubernetes` makes it look like a package from core kubernetes. How about this name? This is what calls its k8s helpers package, so maybe I just like it. :) 
```suggestion
	"persistent-volume-migrator/internal/k8sutil"
```

Reply:
Sure, Sounds good